client/lib/memoize-last/README.md

```diff
@@ -0,0 +1,33 @@
+memoize-last
```

Comment:
sounds a lot like https://www.npmjs.com/package/memoize-one :)

Reply:
Hah, of course there's already an npm library for that :) Let me know if you'd like me to switch over to that one.

Reply:
`lodash.memoize` and our `createSelector` are both very close and could be used to implement `memoizeLast`. But they both have a limitation: they maintain an internal `cache` that has a `Map`-like interface and the arguments need to be converted to a cache key. That limits the types that the arguments can have.

`memoizeLast` doesn't have a cache and the arguments can be anything.

Reply:
> `memoizeLast` doesn't have a cache and the arguments can be anything.

Yes, I think that's one of the main features here; in these situations it's very useful to be able to do a shallow comparison of the argument list, without worrying what those arguments actually are.

Let's stick with `lib/memoize-last` for now, then. If at some point we need more, we can always switch to `memoize-one`, which apparently handles some things (such as `this` changing) a lot better.